libriscan/biblios/templates/biblios/base.html

```diff
@@ -8,8 +8,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>{% block title %}Libriscan{% endblock %}</title>
   {% htmx_script %}
-  <!-- Tailwind - daisyUI CSS -->
-  <link href="{% static 'css/output.css' %}" rel="stylesheet" type="text/css" />
+  <!-- daisyUI CSS CDN -->
+  <link href="https://cdn.jsdelivr.net/npm/daisyui@5" rel="stylesheet" type="text/css" />
+  <script src="https://cdn.jsdelivr.net/npm/@tailwindcss/browser@4"></script>
```

Comment:
Please pull stage and update your branch.
Follow ReadMe to generate output.css for your newly added css.

Your PR shouldn't revert Abdell's change and put back cdn

Reply:
resolved